databricks/koalas/base.py

```diff
@@ -391,7 +398,11 @@ def hasnans(self):
         sdf = self._internal.spark_frame
         scol = self.spark.column
 
-        return sdf.select(F.max(scol.isNull() | F.isnan(scol))).collect()[0][0]
+        if isinstance(self.spark.data_type, DoubleType):
```

Comment:
Shall we add `FloatType` as Spark can have `Float.NaN` as well?

Reply:
Thanks, @ueshin